packages/stylelint/README.md

```diff
@@ -0,0 +1,65 @@
+# Neutrino stylelint Middleware
```

Comment:
Need a `docs/` copy of this too.. :-)

Reply:
Also a reference from `docs/SUMMARY.md`

Reply:
Yup - I'll wait till things gel a bit, then add it.

Reply:
Added!